src/components/Slider/__stories__/Slider.stories.mdx

```diff
@@ -0,0 +1,219 @@
+
+<!--- Component documentation -->
+
+# Slider
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+
+Please write here the component description
+
+<Canvas>
+  <Story name="Overview" args={{}}>
+    {sliderTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable of={Slider} />
+
+## Usage
+
+<UsageGuidelines guidelines={["Usage guideline 1", "Usage guideline 2", "Usage guideline 3"]} />
+<Tip title="Not what you were looking for?">Tip content</Tip>
+
+## Variants
+
+### Sizes
+
+Sizes small/medium/large are available.
+
+<Canvas>
+  <Story name="Sizes">
+    <Slider size={Slider.sizes.SMALL} valueDefault={12} />
+    <Slider size={Slider.sizes.MEDIUM} valueDefault={24} />
+    <Slider size={Slider.sizes.LARGE} valueDefault={35} />
+  </Story>
+</Canvas>
+
+### Ranged Slider
+
+Slider can define range instead of single value
+
+<Canvas>
+  <Story name="Ranged">
+    <Slider size={Slider.sizes.SMALL} isRange={true} />
+    <Slider size={Slider.sizes.MEDIUM} isRange={true} valueDefault={[12, 55]} />
+    <Slider size={Slider.sizes.LARGE} isRange={true} valueDefault={[25, 32]} />
+  </Story>
+</Canvas>
+
+
+### Colors
+
+Color Modes primary/positive/negative are available.
+
+<Canvas>
+  <Story name="Colors">
+    <Slider color={Slider.colors.POSITIVE} valueDefault={34} size={Slider.sizes.MEDIUM} />
+    <Slider color={Slider.colors.NEGATIVE} isRange={true} valueDefault={[12, 55]} size={Slider.sizes.MEDIUM} />
+    <Slider color={Slider.colors.PRIMARY} valueDefault={12} size={Slider.sizes.MEDIUM} />
+  </Story>
+</Canvas>
+
+### Disabled
+
+Slider can be disabled.
+
+<Canvas>
+  <Story name="Disabled">
+    <Slider disabled={true} valueDefault={24} color={Slider.colors.POSITIVE} size={Slider.sizes.MEDIUM} />
+    <Slider disabled={true} color={Slider.colors.NEGATIVE} size={Slider.sizes.MEDIUM} />
+    <Slider disabled={true} isRange={true} valueDefault={[12, 55]} color={Slider.colors.PRIMARY} size={Slider.sizes.MEDIUM} />
+  </Story>
+</Canvas>
+
+### With labels
+
+Indicate selection at Label, Add Prefix and/or Postfix Icons or Labels
+
+<Canvas>
+  <Story name="WithLabels" decorators={[VerticalStories]}>
+    <Slider indicateSelection={true} valueDefault={12} size={Slider.sizes.SMALL} />
+    <Slider indicateSelection={true} isRange={true} valueDefault={[12, 55]} size={Slider.sizes.SMALL} />
+    <Slider prefix={{ icon: Sound }} indicateSelection={true} valueDefault={70} size={Slider.sizes.SMALL} />
+    <Slider prefix={{ icon: Video }} postfix={{ icon: Sound }} valueDefault={45} size={Slider.sizes.MEDIUM} />
+    <Slider prefix="Vol" indicateSelection={true} valueDefault={0} size={Slider.sizes.LARGE} />
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
```

Comment:
Same here. If we don't have any requests from Evigeny for dos and don'ts we can remove this section

Reply:
Evgeniy working on it.